src/ontology/mondo-ingest.Makefile

```diff
@@ -128,6 +128,24 @@ $(COMPONENTSDIR)/icd10who.owl: $(TMPDIR)/icd10who_relevant_signature.txt | compo
 		remove -T config/properties.txt --select complement --select properties --trim true \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/icd10who.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/icd10who.owl -o $@; fi
 
+.PHONY: component-download-gard.owl
+component-download-gard.owl: | $(TMPDIR)
+	if [ $(MIR) = true ] && [ $(COMP) = true ]; then $(ROBOT) merge -I https://github.com/monarch-initiative/gard/releases/latest/download/gard.owl \
+	annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) -o $(TMPDIR)/$@.owl; fi
+
+$(COMPONENTSDIR)/gard.owl: $(TMPDIR)/gard_relevant_signature.txt | component-download-gard.owl
+	if [ $(COMP) = true ]; then $(ROBOT) remove -i $(TMPDIR)/component-download-gard.owl.owl --select imports \
+		rename --mappings config/property-map-1.sssom.tsv --allow-missing-entities true \
```

Comment:
Why is this step necessary?

Reply:
This is also related to these other accidentally standardized operations (your other [comment](https://github.com/monarch-initiative/mondo-ingest/pull/299#discussion_r1187182707)) on the `$(COMPONENTSDIR/*.owl` goals. In the case of these renames though... they're not needed for the GARD ingest.

But I think we might want to actually keep this step for all of our ingests, just in case in the future one of these predicates that we want to rename actually slips in. Some of these renames appear to be very specific, but some of them are general and I imagine could show up in future ingests (especially if the person updating those ingests is not you or I), e.g.:
```tsv
subject_id	object_id
http://www.w3.org/2004/02/skos/core#prefLabel	http://www.w3.org/2000/01/rdf-schema#label
http://www.w3.org/2004/02/skos/core#altLabel	oboInOwl:hasExactSynonym
```